src/modules/Thread/components/ThreadList/ThreadListItemContent.scss

```diff
@@ -48,9 +48,6 @@
     align-items: flex-start;
 
     .sendbird-thread-list-item-content__middle__body-container {
-      .sendbird-thread-list-item-content__middle__message-item-body {
```

Comment:
no issue in desktop?

Reply:
yes, it seems fine

<img width="333" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/46333979/b9cc6d60-b281-4ea5-bd44-121f7db99863">
<img width="329" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/46333979/0f219c7d-bb4d-412c-8f56-902f0f419590">